### PR TITLE
bpo-36543: What's new: Document how to replace xml.etree.cElementTree

### PR DESCRIPTION
**Pull request by hroncok**
This change does not need a changelog entry.

https://bugs.python.org/issue36543

### CODE REVIEW

File: Doc/whatsnew/3.9.rst [hunk 1]
```diff
@@ -666,7 +666,11 @@ Removed
   module have been removed.  They were deprecated in Python 3.2.
   Use ``iter(x)`` or ``list(x)`` instead of ``x.getchildren()`` and
   ``x.iter()`` or ``list(x.iter())`` instead of ``x.getiterator()``.
-  The ``xml.etree.cElementTree`` module has been removed.
+  The ``xml.etree.cElementTree`` module has been removed,
+  use the ``xml.etree.ElementTree`` module instead.
```
Comment (serhiy-storchaka) [inline Doc/whatsnew/3.9.rst]: We can add a reference to an existing module (but only one per paragraph, so the second mention should not be changed).
```suggestion
  use the :mod:`xml.etree.ElementTree` module instead.
```

Comment (hroncok) [reply Doc/whatsnew/3.9.rst]: We can (and we should). Thanks.

**Review by serhiy-storchaka — APPROVED** [review]
:+1:

**Comment by serhiy-storchaka** [comment]
Thank you for your contribution @hroncok.

**Pull request merged**